src/user_guide/singularity.ipynb

```diff
@@ -736,6 +736,51 @@
     "### Other `singularity exec` options"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "You can use parameter `entrypoint` to execute `interpreter`, `entrypoint` used to set up the primary command for the container or to define a default shell or interpreter to use for running scripts\n",
```

Comment:
You can use parameter `entrypoint` to change the environment in which the interpreter will be executed. An `entrypoint` is usually the name of a shell script that defines a number of environment variables and run the command given by the command line parameters in such a way that the current process is replaced by it,

```shell
#!/bin/bash

set -o errexit
set -o pipefail
set -o nounset

export MY_ENV=MY_VALUE

exec "$@"
```

but it can also be other commands such as `micromamba run -n ENVIRONMENT`.

For exmple, .....